index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<head>
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css" />
+  <link
+    rel="stylesheet"
+    type="text/css"
+    href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+  />
+  <!-- Google tag (gtag.js) -->
+  <script
+    async
+    src="https://www.googletagmanager.com/gtag/js?id=G-F55ED7P0L4"
+  ></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag("js", new Date());
+
+    gtag("config", "G-F55ED7P0L4");
+  </script>
+</head>
+<body>
+  <header class="top-bar">
+    <div>
+      <a href="/" class="pglogo">
+        <img src="logo.png" class="logo-icon" alt="로고" />
+      </a>
+      <a href="/login.html" class="login-btn">
+        <img src="loginbt.png" class="login-icon" alt="로그인" />
+      </a>
+    </div>
+  </header>
+
+  <section class="section2">
+    <div class="section2-wrap">
+      <div class="section-text">
+        <h1>일상의 모든 물건을<br />거래해 보세요</h1>
```

Comment:
저는 시각적인 효과를 위해 br 태그를 사용하는 걸 선호하지 않습니다. (html은 의미에 집중하고 스타일을 다루지 않는다)
지금은 허용할 수 있는 범위지만, 길이 제한이 필요한 경우 css 스타일로 제어하거나, span으로 래핑해서 명시적으로 단락을 구분하는 스타일로 작성하는 걸 연습해보시면 좋을 것 같습니다.

```suggestion
        <h1><span>일상의 모든 물건을</span><span>거래해 보세요</span></h1>
```